example/index.tsx

```diff
@@ -6,26 +6,27 @@ import {CustomNodeWidget} from "./components/widgets/CustomNodeWidget";
 import {makeStyles} from "@material-ui/core";
 import CustomLinkWidget from "./components/widgets/CustomLinkWidget";
 import BG from "./components/assets/svg/bg-dotted.svg";
+import { colorGreen, colorRed } from "./components/assets/styles/constants";
 
 const useStyles = makeStyles(_ => ({
-    main: {
-        position: 'absolute',
-        height: '100%',
-        width: '100%',
-    },
-    canvasBG: {
-        backgroundImage: `url(${BG})`
-    }
-}))
+  main: {
+    position: 'absolute',
+    height: '100%',
+    width: '100%',
+  },
+  canvasBG: {
+    backgroundImage: `url(${BG})`
+  }
+}));
 
 const App = () => {
     const classes = useStyles();
 
-    const node1 = new MetaNode('1', 'node1', 'default', new Position(250, 100),
-        new Map(Object.entries({color: 'rgb(0,192,255)'})))
+    const node1 = new MetaNode('1', 'Node 1', 'default', new Position(250, 100),
+        new Map(Object.entries(colorGreen)))
```

Comment:
I would like this to be in the theme with theme provider like MUI

Reply:
@zsinnema if i add these variants to theme, then it limits us to use only specific variants like green, orange etc. I believe this should not be a restricted option, user should have an option to choose node color options from the project , also whatever color options he/she wants not just limited to the set theme options.

Reply:
I mean create an "application specific" theme and use that theme. If the app wants orange nodes then make if in the theme orange. For variants we can use this: https://mui.com/material-ui/customization/theme-components/#creating-new-component-variants